### PR TITLE
Add GitHub action

### PR DESCRIPTION
**Pull request by OleksandRomaniuk**

### CODE REVIEW

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+
+      - name: Start mongodb
+        run: docker run -d --name mongo -p 27017:27017 mongo:latest
```
Comment (VolodymyrM27) [inline .github/workflows/github-actions.yml:30]: Why are you doing this? Your project will build without connecting to mongoDB.

File: .github/workflows/github-actions.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
```
Comment (VolodymyrM27) [inline .github/workflows/github-actions.yml:20]: Added  `distribution `

     `   with:
          distribution: 'oracle'
          java-version: '17'`

File: .github/workflows/github-actions.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "*" ]
+
+  workflow_dispatch:
+
+jobs:
+  docker:
```
Comment (VolodymyrM27) [inline .github/workflows/github-actions.yml:11]: Why is the whole actions called a docker, even though it does more than that?

File: .github/workflows/github-actions.yml (hunk 2 above)
Comment (OleksandRomaniuk) [reply .github/workflows/github-actions.yml:20]: yes you are right - thats why my first action failed

**Review by ajax-gnatiuk-s — APPROVED** [review]

File: .github/workflows/github-actions.yml (hunk 3 above)
Comment (OleksandRomaniuk) [reply .github/workflows/github-actions.yml]: I rename this , but lets discuss on meeting

**Pull request merged**